public/specifications/monitoring/schemas/status-schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$id": "https://github.com/SRGSSR/pillarbox-documentation/blob/main/Specifications/status-schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "data": {
+      "type": "object",
+      "description": "Data associated with the status event, containing details about the current state of playback, network, and device.",
+      "properties": {
+        "airplay": {
+          "type": "boolean",
+          "description": "Indicates whether AirPlay is currently active."
+        },
+        "bandwidth": {
+          "type": "integer",
+          "description": "The available bandwidth in bits per second."
+        },
+        "bitrate": {
+          "type": "integer",
+          "description": "The bitrate of the content being played in bits per second."
+        },
+        "buffered_duration": {
+          "type": "number",
+          "description": "The duration of the content currently available in the buffer, in milliseconds."
+        },
+        "duration": {
+          "type": "number",
+          "description": "The content duration, as retrieved from the playlist, in milliseconds."
+        },
+        "playback_duration": {
+          "type": "number",
+          "description": "The duration of the playback session in milliseconds, measured in wall-clock time."
+        },
+        "position": {
+          "type": "number",
+          "description": "The current player position relative to the beginning of the playlist, in milliseconds. Negative values are admitted."
+        },
+        "position_timestamp": {
+          "type": "integer",
+          "description": "The current player timestamp as retrieved from the playlist, in Unix milliseconds. This field may be omitted if not available."
+        },
+        "stall": {
+          "type": "object",
+          "description": "Information about playback stalls experienced during the session.",
+          "properties": {
+            "count": {
+              "type": "integer",
+              "description": "The total number of stalls experienced during the session."
+            },
+            "duration": {
+              "type": "number",
+              "description": "The total duration of stalls in milliseconds, measured in wall-clock time."
+            }
+          },
+          "required": ["count", "duration"]
+        },
+        "stream_type": {
+          "type": "string",
+          "enum": ["On-demand", "Live"],
+          "description": "The type of stream being played, either 'On-demand' or 'Live'."
```

Comment:
Could we have a single format whether `uppercase` or `lowercase` to have some consistency? Today we have **lowercase**, **UPPERCASE** and **Capitalize**.

Reply:
This could be fixed in a separated PR.

Reply:
I would vote for capitalized then, as it is the most readable option  when the data is represented (e.g. iPhone intstead of ugly iphone or IPHONE).

Of course this means that if you seek for perfect consistency START should be renamed as Start and ERROR as Start. 

I implicitly assumed that the current state was rather that "what can be displayed on a chart is capitalized, other internal stuff like event names are uppercase". But of course this was never formally discussed.

Reply:
I won't argue about the ugliness of the Macintosh iPhone. However while reviewing this PR I came across:

https://github.com/SRGSSR/pillarbox-documentation/blob/ee1705b6d674517a2f5b5a85275dd56101234a59/public/specifications/monitoring/MONITORING.md?plain=1#L293

Followed by

https://github.com/SRGSSR/pillarbox-documentation/blob/ee1705b6d674517a2f5b5a85275dd56101234a59/public/specifications/monitoring/MONITORING.md?plain=1#L343

I'm not against capitalizing events because it makes them stand out. Or to keep the case for all values that are returned by the system or brands. But for everything else, I can't find any logic that would make me think it's a good idea to press the shift key.

Reply:
Depends what goal you want to achieve. If the rule is "spare your shift key, except for values provided by the system or brands", then every field format should be decided based on this rule.

In general I prefer the rule "readable, displayable and consistent", and to achieve maximum consistency the rule should be pretty simple IMHO, so that the format of each field is straightforward to determine. This is why I proposed a single "capitalized" rule, but I am comfortable with other rules, as long as we can write how a field format should be determined with a single easy to remember sentence, not as a rule with a series of exceptions.

Reply:
We'll iterate on the formatting on another PR. This has already been discussed elsewhere.